daml-lf/spec/daml-lf-1.rst

```diff
@@ -3436,6 +3532,20 @@ updates.
   FIXME: https://github.com/digital-asset/daml/issues/2256
     Handle contract IDs
 
+Rounding Mode Rank
+..................
+
+To simplify the definition of rounding mode ordering, we define an auxiliary definion that maps them to integers:
+
```

Comment:
Add a comment saying that there is no guarantee this `roundingModeRank` is stable across different version of this specification, just the resulting ordering it is.

Reply:
a36fa5069aebd39914f4f676723442aefe83e4ec